static/js/filter_country.js

```diff
@@ -17,6 +16,9 @@ const filterCriteria = (function() {
 				toggleDisable("country",online_dom.value)
 			}
 		}
+		if(querySt("language")){
```

Comment:
here with double quotes and in the next line with single quotes...
start normalizing this...